tests/test_observables.py

```diff
@@ -414,6 +425,16 @@ def test_dnstwist(self, mock_get=None, mock_post=None):
 
         self.assertEqual(report.get("success", False), True)
 
+    def test_zoomeye(self, mock_get=None, mock_post=None):
+        report = robtex.Robtex(
```

Comment:
:eyes: you made a mistake here

Reply:
My Bad!

Reply:
will merge your PR after u have fixed this